src/envo/envo-edit.owl

```diff
@@ -42164,7 +42164,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03501298> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03501299> (charcuterie shop)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501299> "A delicatessen-style shop that prepares and sells preserved meat products."@en)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03501299> "A delicatessen-style shop that prepares and sells preserved meat products."@en)
```

Comment:
@ddooley I think definition style should be: A delicatessen which prepares and sells preserved meat products.

Reply:
Good!